tftrt/examples/image-classification/image_classification.py

```diff
@@ -65,23 +72,32 @@ def run(frozen_graph, model, data_files, batch_size,
     data_files: List of TFRecord files used for inference
     batch_size: int, batch size for TensorRT optimizations
     num_iterations: int, number of iterations(batches) to run for
+    num_warmup_iterations: int, number of iteration(batches) to exclude from benchmark measurments
+    use_synthetic: bool, if true run using real data, otherwise synthetic
+    display_every: int, print log every @display_every iteration
+    run_calibration: bool, run using calibration or not (only int8 precision)
+	mode: validation - using estimator.evaluate with accuracy measurments,
+          benchmark - using estimator.predict
     """
```

Comment:
Please add a check after this line to ensure `mode` has one of the two values: `validation` or `benchmark`. Otherwise raise an exception.

Reply:
Similarly in other functions that use `mode` throughout the script.